sklearn/ensemble/_gradient_boosting.pyx

```diff
@@ -219,7 +219,6 @@ def predict_stages(np.ndarray[object, ndim=2] estimators,
                     tree.nodes, tree.value,
                     scale, k, K, X.shape[0], X.shape[1],
                     <float64 *> (<np.ndarray> out).data)
-                ## out += scale * tree.predict(X).reshape((X.shape[0], 1))
```

Comment:
I think this is a reminder that `_predict_regression_tree_inplace_fast_dense` is the same as `out += scale * tree.predict(X).reshape((X.shape[0], 1))`.

We can make the comment more explicit:

```python
# _predict_regression_tree_inplace_fast_dense is the same as
# out += scale * tree.predict(X).reshape((X.shape[0], 1))
```

Reply:
Thank you @thomasjpfan 
Yes, the results would be the same, but the time for classifying would be different, I think it would be better to have both as two different methods for prediction.

Reply:
> but the time for classifying would be different, I think it would be better to have both as two different methods for prediction.

I do not understand what you mean to have both. `_predict_regression_tree_inplace_fast_dense` is a drop in replacement for `out += scale ...` that is faster.

Currently this PR is for removing the comment. The comment is still useful to have so that a reader can quickly understand what `_predict_regression_tree_inplace_fast_dense` is doing.

Reply:
In this case, we could continue with the detailed comment you mentioned.